packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -165,7 +166,9 @@ export const getComponentCss = (
       padding: '4px 0', // for slide focus outline
       margin: '-4px 0', // for slide focus outline
       '&__track': {
-        cursor: 'grab',
+        ...(hasNavigation && {
```

Comment:
Why don't we use the already existing css modifiers that splide provides? 
`splide--draggable` and `splide__track--draggable`?